### PR TITLE
Clear state on encrypt password page Issue-76

### PR DESCRIPTION
**Pull request by SumitKumar-17**
This is with the Issue-76 Clear state on encrypt password page,when the page reloads ,now the image .Now the EncryptPassword component does not  loads the previous encrypted image from state, on page load this should be cleared.

### CODE REVIEW

File: frontend/src/services/password-thunk.js [hunk 1]
```diff
@@ -134,3 +134,8 @@ export const uploadBackupThunk = createAsyncThunk(
     }
   }
 );
+
+export const clearEncryptData = createAsyncThunk(
```
Comment (frankhart2018) [inline frontend/src/services/password-thunk.js]: This can be a reducer, does not need to be an async thunk

**Review by frankhart2018 — CHANGES_REQUESTED** [review]

File: frontend/src/services/password-thunk.js (hunk 1 above)
Comment (SumitKumar-17) [reply frontend/src/services/password-thunk.js]: should i create it  as 

export const clearEncryptData = createAction('password/clearEncryptData');  
 in the source code .

Comment (frankhart2018) [reply frontend/src/services/password-thunk.js]: Nope right here: https://github.com/frankhart2018/primistore/blob/b77ff2dc877c0b4e1f0f70b2f217179efabb803f/frontend/src/reducers/password-reducer.js#L41. Here's an example: https://github.com/frankhart2018/cs-5610-404-not-found/blob/08de309606e095cd1937f24185709968fffc8bf3/frontend/src/reducers/file-reducer.js#L17.

**Comment by SumitKumar-17** [comment]
I suppose that you are aksing for this in the password-reducer.js . i have made the suggested change in the commits and squashed my commits into one.
reducers: {
    clearEncryptedData: (state) => {
      state.encryptedData = getZeros2DArray(ROWS, COLS);
    }
  },

**Review by frankhart2018 — APPROVED** [review]
Looks good.

**Pull request merged**